### ISSUE TITLE
Error while retrieving favicons

### ISSUE THREAD

**Issue by jankal** [pimlie/nuxt-rfg-icon#5]
```
  plugin:nuxt-rfg-icon Retrieving favicons from realfavicongenerator api +0ms
  plugin:nuxt-rfg-icon error while communicating with rfg api ReferenceError: head is not defined
    at axios.post.then.then.then (/c/Users/alexa/Projekte/zeraton-hp/node_modules/nuxt-rfg-icon/index.js:289:75)
    at <anonymous> +17s
```
How can this be fixed?

My config:
```js
[
    ['nuxt-rfg-icon', {
      static: true,
      staticPath: '/_favicons/',
      masterPicture: 'static/icon.png',
      rfg: require('./favicon.json')
    }]
]
```

My favicon config (`favicon.json`):
```json
{
	"design": {
		"ios": {
			"pictureAspect": "backgroundAndMargin",
			"backgroundColor": "#ffffff",
			"margin": "14%",
			"assets": {
				"ios6AndPriorIcons": false,
				"ios7AndLaterIcons": false,
				"precomposedIcons": false,
				"declareOnlyDefaultIcon": true
			}
		},
		"desktopBrowser": {},
		"windows": {
			"pictureAspect": "noChange",
			"backgroundColor": "#da532c",
			"onConflict": "override",
			"assets": {
				"windows80Ie10Tile": false,
				"windows10Ie11EdgeTiles": {
					"small": false,
					"medium": true,
					"big": false,
					"rectangle": false
				}
			}
		},
		"androidChrome": {
			"pictureAspect": "noChange",
			"themeColor": "#ffffff",
			"manifest": {
				"display": "standalone",
				"orientation": "notSet",
				"onConflict": "override",
				"declared": true
			},
			"assets": {
				"legacyIcon": false,
				"lowResolutionIcons": false
			}
		},
		"safariPinnedTab": {
			"pictureAspect": "silhouette",
			"themeColor": "#5bbad5"
		}
	},
	"settings": {
		"compression": 2,
		"scalingAlgorithm": "Mitchell",
		"errorOnImageTooSmall": false,
		"readmeFile": false,
		"htmlCodeFile": false,
		"usePathAsIs": false
	}
}
```

When I set this up earlier this year, everything still worked.

**Comment by pimlie** [comment]
This has been fixed now in v0.4.3. I am currently adding some test cases as this bug was caused by a stupid mistake, so hopefully when I have the tests added it wont happen again ;)

**Comment by jankal** [comment]
@pimlie Thanks!
